DevCycle.SDK.Server.Common/Model/Local/Variable.cs

```diff
@@ -146,7 +142,39 @@ public static TypeEnum DetermineType(T variableValue)
             {
                 Console.WriteLine(variableValue);
                 Console.WriteLine(e);
-                throw e;
+                throw;
+            }
+        }
+
+        private static bool IsNumericType(Type type)
```

Comment:
Can we please add unit tests for this change.

Reply:
The test harness already handles this.

Reply:
The test harness does not check all these types, the test harness is for E2E tests, not unit tests.